client/my-sites/plans/main.jsx

```diff
@@ -2,31 +2,36 @@
  * External dependencies
  */
 import { connect } from 'react-redux';
+import i18n from 'i18n-calypso';
```

Comment:
Let's use the [localize](https://github.com/Automattic/i18n-calypso#localize) higher order component:
```
import { localize } from 'i18n-calypso';
```